### PR TITLE
Changed docker default branch

### PR DESCRIPTION
**Pull request by eorituz**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -12,7 +12,7 @@ FROM python:3.7 as builder
 
 # use --build-arg RAIDENVERSION=v0.0.3 to build a specific (tagged) version
 ARG REPO=raiden-network/raiden
-ARG RAIDENVERSION=master
+ARG RAIDENVERSION=develop
```
Comment (LefterisJP) [inline docker/Dockerfile]: @ulope Shouldn't we have two different tags for the docker files? A stable/latest and a dev one?

Comment (eorituz) [reply docker/Dockerfile]: https://cloud.docker.com/u/raidennetwork/repository/docker/raidennetwork/raiden

There is latest which gets build every time we release a new RC
There is nightly which will be built every night (in sync with the nightly binaries)
There is unstable which gets build with every commit in develoü

Comment (ulope) [reply docker/Dockerfile]: As long as the file in the respective branch points to the correct one everything is fine.

Ideally we should switch this to use the local checkout and not clone the repo again (as we did with `build.Dockerfile`).

Comment (ulope) [reply docker/Dockerfile]: I made an issue to track this: #4483

**Pull request merged**